packages/ui/src/modal/bid/BidModalRenderer.tsx

```diff
@@ -219,15 +213,18 @@ export const BidModalRenderer: FC<Props> = ({
 
   useEffect(() => {
     if (bidAmount !== '') {
-      const bid = parseUnits(bidAmount, wrappedBalance?.decimals)
+      const bid = parseUnits(
+        `${Number(bidAmount)}`,
+        wrappedBalance?.decimals || 18
+      )
 
-      if (!wrappedBalance?.value || wrappedBalance?.value.lt(bid)) {
+      if (!wrappedBalance?.value || wrappedBalance?.value < bid) {
         setHasEnoughWrappedCurrency(false)
-        const wrappedAmount = wrappedBalance?.value || constants.Zero
-        const amountToWrap = bid.sub(wrappedAmount)
-        setAmountToWrap(formatBN(bid.sub(wrappedAmount), 5))
+        const wrappedAmount = wrappedBalance?.value || 0n
```

Comment:
Just want to be sure that 0n is intended and not a typo.

Reply:
Yep, `0n` is the BigInt equivalent of the number `0`